easybuild/easyconfigs/v/VTK/VTK-9.3.0-foss-2023a.eb

```diff
@@ -57,7 +59,11 @@ configopts += "-DPython3_INCLUDE_DIR=$EBROOTPYTHON/include/python%(pyshortver)s
 configopts += "-DPython3_LIBRARY=$EBROOTPYTHON/lib/libpython%(pyshortver)s.so "
 # Other
 configopts += "-DVTK_USE_MPI=ON "
-configopts += "-DCMAKE_INSTALL_LIBDIR=lib"
+configopts += "-DCMAKE_INSTALL_LIBDIR=lib "
+configopts += "-DVTK_QT_VERSION=5 "
+configopts += "-DQt5_DIR=$EBROOTQT5 "
+configopts += "-DVTK_MODULE_ENABLE_VTK_GuiSupportQt=YES "
+configopts += "-DVTK_MODULE_ENABLE_VTK_ViewsQt=YES"
```

Comment:
@pavelToman Can you open a separate PR for the changes being made here to the existing easyconfig for VKT 9.3.0, so it stands out a bit better?

Reply:
PR: https://github.com/easybuilders/easybuild-easyconfigs/pull/21221

Reply:
see #21221